src/modules/displaymanager/main.py

```diff
@@ -544,6 +544,9 @@ class DMlightdm(DisplayManager):
     name = "lightdm"
     executable = "lightdm"
 
+    # Can be overridden in the .conf file
+    preferred_greeters = ["lightdm-greeter.desktop"]
```

Comment:
Since `lightdm-greeter` is debian specific, maybe `lightdm-gtk-greeter` is a more reasonable default?

Reply:
I've picked "empty list" as default so it doesn't have **any** preferred one, which falls back to the older behavior (except now it's deterministic by alphabet).